io.github.andrewmkiss.Demeter.yml

```diff
@@ -0,0 +1,190 @@
+app-id: io.github.andrewmkiss.Demeter
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: dathena
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --filesystem=host
+
+modules:
+  - name: hdf4_lib
+    disabled: false 
+    buildsystem: simple
+    no-autogen: true
+    build-commands:
+      - 'hdf4_lib/install.sh'
```

Comment:
why?

Reply:
yeah...good question. working on doing this correctly